plotly/tools.py

```diff
@@ -364,17 +369,10 @@ def embed(file_owner_or_url, file_id=None, width="100%", height=525):
         pass
     if _ipython_imported:
         if file_id:
-            plotly_domain = (
-                session.get_session_config().get('plotly_domain') or
-                get_config_file()['plotly_domain']
-            )
-            url = "{plotly_domain}/~{un}/{fid}".format(
-                plotly_domain=plotly_domain,
-                un=file_owner_or_url,
-                fid=file_id)
+            return PlotlyDisplay(file_owner_or_url, file_id, share_key, width,
+                                 height)
         else:
-            url = file_owner_or_url
-        return PlotlyDisplay(url, width, height)
+            return PlotlyDisplay(file_owner_or_url, width, height)
```

Comment:
The `file_id` argument is required as defined in `PlotlyDisplay`. Does this work?